apps/backend/src/services/audit_service.py

```
@@ -0,0 +1,443 @@
+            partial_operations = status_counts.get("partial", 0)
+            timeout_operations = status_counts.get("timeout", 0)
+
+            # Calculate rates
+            success_rate = (successful_operations / total_operations * 100) if total_operations > 0 else 0
+            failure_rate = (failed_operations / total_operations * 100) if total_operations > 0 else 0
+
+            # Get cache metrics
+            cache_hit_query = (
+                select(func.count())
+                .select_from(base_query.subquery())
+                .where(DataCollectionAudit.cache_hit == True)
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `DataCollectionAudit.cache_hit:` for truth checks

Replace with `DataCollectionAudit.cache_hit`

(E712)